### ISSUE TITLE
Default/Premade Lists

### ISSUE THREAD

**Issue by SamJones04** [tachiyomiorg/tachiyomi#8893]
### Describe your suggested feature

It would be cool to have an option to choose a list template from tachiyomi without coming up with my own.

The normal one I use is:

> Reading
> Completed
> On-hold
> Dropped
> Plan-to-Read

Would It be possible to make a guide or premade template made specifically for Tachiyomi that works within its limits? My Current List gets warning errors about splitting it up into smaller reading lists.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.14.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Ihccato** [comment]
isn't that what categories are for? you can create a category and place manga in there.

**Comment by SamJones04** [comment]
> isn't that what categories are for? you can create a category and place manga in there.

Yeah that's what I mean, I get errors because my reading list is too long so it would be nice to have a guide or premade categories telling how many u should have per source or how many max you should have. It's pretty hard work going through 300+ Series without any idea how to order them.
